pom.xml

```diff
@@ -26,7 +26,6 @@
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
         <jenkins.version>2.150.3</jenkins.version>
-        <scm-api.version>2.6.3</scm-api.version>
         <hamcrest.version>2.2</hamcrest.version>
         <useBeta>true</useBeta>
         <jcasc.version>1.35</jcasc.version>
```

Comment:
see https://github.com/jenkinsci/bom/pull/172

Reply:
So, should we wait on this change until that pr is resolved?

Reply:
No, just FYI.